upf/upf_pfcp_api.c

```diff
@@ -728,7 +728,6 @@ handle_f_teid (upf_session_t * sx, upf_main_t * gtm, pfcp_pdi_t * pdi,
 	{
 	  vec_add2 (*created_pdr_vec, created_pdr, 1);
 	  memset (created_pdr, 0, sizeof (*created_pdr));
-	  SET_BIT (grp->fields, SESSION_ESTABLISHMENT_RESPONSE_CREATED_PDR);
```

Comment:
This removes the only user of the `grp` function argument. The argument should also be removed.

Reply:
fixed, thanks for note